File: LeetCode_1748/Main.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var sumOfUnique = function(nums) {
    const cnt = [];
    for (let num of nums) {
        cnt[num] = cnt[num] == undefined ? 1 : cnt[num] + 1;
    }
    
    let sum = 0;
    for (let i = 1; i <= 100; i++) {
        sum = cnt[i] == 1 ? sum + i : sum;
    }
    return sum;
};

File: LeetCode_1748/Main.py
class Solution(object):
    def sumOfUnique(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        cnt = [0 for _ in range(101)]
        for num in nums:
            cnt[num] += 1
        return sum(i for i in range(1, 101) if cnt[i] == 1)